bogenliga/src/app/modules/spotter/components/authentication/authentication.component.ts

```diff
@@ -0,0 +1,48 @@
+import {Component, OnInit} from '@angular/core';
+import {ButtonSize} from '@shared/components';
+
+// new CM
+// import {Component, OnInit} from '@angular/core';
+import {TabletSessionDO} from '@sportjahresplan/types/tablet-session-do.class';
+import {BogenligaResponse} from '@shared/data-provider';
+import {ActivatedRoute, Router} from '@angular/router';
+import {TabletSessionProviderService} from '@sportjahresplan/services/tablet-session-provider.service';
+
+
+
+@Component({
+  selector: 'bla-authentication',
+  templateUrl: './authentication.component.html',
+  styleUrls: ['./authentication.component.scss']
+})
+
+export class AuthenticationComponent implements OnInit {
+
+  constructor(private tabletSessionService: TabletSessionProviderService, private route: ActivatedRoute, private router: Router) {}
+
+  sessions: Array<TabletSessionDO>;
+  currentSession: TabletSessionDO;
+  tokens = [];
+  AccessTokenInput: string;
+  public ButtonSize = ButtonSize;
```

Comment:
setzt doch hier direkt auch buttonSize.large statt im HTML?

Reply:
Hi, habe mich hier an der Implementierung von in login.component.html bzw .ts gehalten. 

Verstehe nicht so ganz was du meinst. :/

Reply:
Ok, dann ists einheitlich. Passt